documentation/index.html.js

```diff
@@ -937,6 +938,29 @@ Expressions
     </p>
     <%= codeFor('heregexes') %>
 
+    <p>
+      <span id="modules" class="bookmark"></span>
+      <b class="header">Modules</b>
+      ES2015 modules are supported in CoffeeScript, with a very similar <code>import</code>
+      and <code>export</code> syntax:
+    </p>
+    <%= codeFor('modules') %>
+    <p>
+      Note that the CoffeeScript compiler <strong>does not resolve modules</strong>; writing an
+      <code>import</code> or <code>export</code> statement in CoffeeScript will produce an
+      <code>import</code> or <code>export</code> statement in the resulting output.
+      It is your responsibility attach another transpiler, such as
+      <a href="https://github.com/google/traceur-compiler">Traceur Compiler</a>,
+      <a href="http://babeljs.io/">Babel</a>&nbsp;or
+      <a href="https://github.com/rollup/rollup">Rollup</a>, to convert this ES2015 syntax into
+      code that will work in your target runtimes.
+    </p>
+    <p>
+      Also note that any file with an <code>import</code> or <code>export</code> statement will
+      be output without a <a href="#lexical-scope">top-level function safety wrapper</a>;
+      in other words, importing or exporting modules will automatically trigger
+      <a href="#usage">bare</a> mode for that file. This is because per the ES2015 spec,
+      <code>import</code> or <code>export</code> statements must occur at the topmost scope.
```

Comment:
There’s no convenient way to support all three. You either have to know you’re using modules _or_ you’re targeting the browser/CommonJS. You can’t even do some kind of check as far as I’m aware, like “if `export` is supported...”. I guess just leave it as I’ve amended it, with the mention of `export`?